src/main/java/org/cactoos/set/Sorted.java

```diff
@@ -58,15 +53,9 @@ public Sorted(final Comparator<T> cmp, final T... array) {
      * @param cmp Comparator
      * @param src An {@link Iterable}
      */
+    @SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
```

Comment:
@victornoel @paulodamaso We probably need to understand why this one required here, but not in `ListOf`

Reply:
@fanifieiev @paulodamaso what do you propose?

Reply:
@victornoel For example, create an issue for `Qulice `project